data/projects/b/basket-swap-stylus-teeeeeo9.yaml

```diff
@@ -3,7 +3,7 @@ name: basket-swap-stylus-teeeeeo9
 display_name: BasketSwap
 description: BasketSwap doesn't compete against existing projects but rather bounds existing fragmented liquidity for the user's convenience. It is the first of its kind in terms of product and leverages Stylus capabilities.
 websites:
-  - https://basketswap.xyz
+  - url: https://basketswap.xyz
 social:
   - twitter: "-"
```

Comment:
Let's not add null fields (eg, "-") rather than include something that isn't a url.